.github/workflows/auto_update.yml

```diff
@@ -35,8 +25,6 @@ jobs:
           now=$(date '+%Y-%m-%d-%H%M%S')
           BRANCH_NAME="db-update-$now"
           git checkout -b "$BRANCH_NAME"
```

Comment:
Something that came to my mind: Do we even need to push a new branch to GitLab? Can't we run the pipeline just on `main`? That could actually simplify the workflow even more.

Reply:
I think that the issue is we don't push to main on the CI but only on branches, I've imported this behavior from other projects. So I think we need to push a branch there. Is it done some other way in other projects ?

Reply:
We do, see https://github.com/osbuild/manifest-db/blob/799deeac2285c0e5e1706e4c3b0e02ea60211b4c/.github/workflows/trigger-gitlab.yml#L49.

It's weird though that `main` on gitlab and on github are out of sync currently. I looked into logs and saw that you were the last one to push into gitlab's `main`?

![image](https://user-images.githubusercontent.com/119861/189615188-c328b792-9420-4217-bfc9-cc9b9900ea9e.png)

So if we stop doing that, then these branches should be in sync (after github actions pass on any new commits). I protected the `main` branch in Gitlab, humans shouldn't be able to push into it anymore.

Reply:
Wow! I was not aware of this push. That must be quite an accidental one!

Reply:
Also, I'd have to create a branch to open my PR on GH. Do you think it's best to move this branch creation on gitlab's side ?